components/list/list-item-nav-mixin.js

```diff
@@ -44,6 +46,26 @@ export const ListItemNavMixin = superclass => class extends ListItemLinkMixin(su
 				--d2l-list-item-content-text-color: var(--d2l-color-ferrite);
 				--d2l-list-item-content-text-decoration: none;
 			}
+			:host([current]) d2l-button-add,
+			:host([_has-current-parent]) [slot="add-top"] d2l-button-add,
+			:host([_next-sibling-current]) [slot="add"] d2l-button-add {
+				--d2l-button-add-hover-focus-line-height: 3px;
+				--d2l-button-add-line-color: var(--d2l-color-celestine);
+				--d2l-button-add-line-height: 3px;
+			}
```

Comment:
An add button can be associated with an item itself, but can also be part of the previous or next list item, which gets complicated with `current`.

For example:
<img width="486" alt="Screenshot 2025-06-05 at 8 37 45 AM" src="https://github.com/user-attachments/assets/4e864741-e27f-4588-a2fb-450d7adb7708" />

This is because the item above has `d2l-button-add` in its `add` slot.

If there is an easy way to target these situations just through css let me know. I was unable to get something working and had to resort to adding properties through js.